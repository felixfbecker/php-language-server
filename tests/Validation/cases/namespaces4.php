<?php
use  \a\b as c;