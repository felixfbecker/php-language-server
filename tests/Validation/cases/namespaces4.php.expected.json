{
    "references": {
        "a\\b": [
            "./namespaces4.php"
        ],
        "a": [
            "./namespaces4.php"
        ]
    },
    "definitions": []
}